### PR TITLE
Feat : Archieve-Note

### PR DESCRIPTION
**Pull request by anuj-kumary**
Added Archive Functionality
    - Archive Note
    - Restore Archive Note
    - Delete Archive Note

To see your archive note click on the archive button of the sidebar
Preview:- https://deploy-preview-5--eagle-notes.netlify.app/

### CODE REVIEW

**Review by romabulani — APPROVED** [review]
- Liked the UI
- Clean code and good to merge

Suggestion : In some files, there is red sign at the end of File, make sure to enter one line at the end of file

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET = नहीं बताना
```
Comment (Rohanmond) [inline .env:1]: thanks for providing this info😂

File: src/pages/Archive/Archive.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+  return (
+    <>
+      <div className='note__container'>
+        <Sidebar />
+        {state.archiveList.map((note) => {
+          const { _id, title, content, timeCreated, backgroundColor } = note;
+          return (
+            <div
+              style={{ backgroundColor: backgroundColor }}
```
Comment (Rohanmond) [inline src/pages/Archive/Archive.jsx:43]: can be written as style={{backgroundColor}}

**Review by Rohanmond — APPROVED** [review]

File: src/pages/Archive/Archive.jsx (hunk 2 above)
Comment (anuj-kumary) [reply src/pages/Archive/Archive.jsx:43]: Sure I'll update this

**Pull request merged**